### PR TITLE
Add opt-in screenshot attachment for crash events captured on Windows/Linux

### PR DESCRIPTION
**Pull request by tustanivsky**
Related to #567

### CODE REVIEW

**Review by bitsandfoxes — COMMENTED** [review]
This looks great!

File: plugin-dev/Source/Sentry/Private/Desktop/SentrySubsystemDesktop.cpp [hunk 1]
```diff
@@ -129,6 +133,19 @@ void SentrySubsystemDesktop::InitWithSettings(const USentrySettings* settings, U
 #endif
 	}
 
+	if(settings->AttachScreenshot)
+	{
+		isScreenshotAttachmentEnabled = true;
+
+		const FString ScreenshotPath = FGenericPlatformOutputDevices::GetAbsoluteLogFilename();
```
[the remark this answers is not in the document]
Comment (tustanivsky) [reply plugin-dev/Source/Sentry/Private/Desktop/SentrySubsystemDesktop.cpp]: Good catch, this is not required here.

File: plugin-dev/Source/Sentry/Private/SentrySubsystem.cpp
[the remark this answers is not in the document]
Comment (tustanivsky) [reply plugin-dev/Source/Sentry/Private/SentrySubsystem.cpp:1]: We'll probably cover this in scope of #575

**Review by bitsandfoxes — APPROVED** [review]
Just to double check: In case the screenshot capture fails to write the file to disk, does the crashhandler deal with it gracefully?

**Comment by bruno-garcia** [comment]
Is this on the docs? Couldn't find any reference to screenshots there

**Pull request merged**